liblwgeom/lwgeom_sfcgal.c

```diff
@@ -136,18 +136,23 @@ ptarray_from_SFCGAL(const sfcgal_geometry_t *geom, int want3d)
 
 	assert(geom);
 
+	int is_3d = sfcgal_geometry_is_3d(geom);
+	int is_measured = sfcgal_geometry_is_measured(geom);
```

Comment:
Requires a ifdef SFCGAL >= 1.3.8

According to [Repology](https://repology.org/project/sfcgal/versions) Ubuntu 20.04 and Debian 10 still use SFCGAL 1.3.6